.github/ISSUE_TEMPLATE/2-Question.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Question
+about: Ask a question.❓
+labels: 'question'
+---
+
+## Prerequisites
```

Comment:
I'm wondering if questions should require environment/prereqs. I think this makes sense for bug reports. Not sure about questions. Thoughts on this @jooohhn?

Reply:
updated